src/resources/views/crud/columns/array_count.blade.php

```diff
@@ -1,24 +1,24 @@
 {{-- enumerate the values in an array  --}}
 @php
     $array = data_get($entry, $column['name']);
-    $suffix = isset($column['suffix']) ? $column['suffix'] : 'items';
+
+    $column['escaped'] = $column['escaped'] ?? false;
+    $column['prefix'] = $column['prefix'] ?? '';
+    $column['suffix'] = $column['suffix'] ?? 'items';
```

Comment:
```suggestion
    $column['suffix'] = $column['suffix'] ?? ' items';
```

Reply:
@pxpm this file is problematic.
By default it had the space before the the word `items`. So we decided to keep it in v4.1.
The worst part is the space was hard-coded not in the suffix, but in the code.

So there is a patch already for 4.2;
https://github.com/Laravel-Backpack/CRUD/pull/3436/files

Reply:
Thanks for clarification. :)